.github/workflows/test.yml

```diff
@@ -8,6 +8,9 @@ on:
     branches: ['production', 'production-staging', 'hotfix/*']
     types: ['opened', 'reopened', 'synchronize']
 
+env:
+  NODE_OPTIONS: "--no-warnings"
```

Comment:
What warnings did we hide?

Reply:
https://github.com/rudderlabs/rudder-sdk-js/actions/runs/6083720619/job/16504180123#step:7:127